build/build.ps1

```diff
@@ -326,7 +326,7 @@ try {
   $ToolsetBuildProj = Join-Path $NuGetPackageRoot "RoslynTools.RepoToolset\$ToolsetVersion\tools\Build.proj"
 
   $vsInstallDir = LocateVisualStudio
-  $MsbuildExe = Join-Path $vsInstallDir "MSBuild\15.0\Bin\msbuild.exe"
```

Comment:
We should only use `Current` if `15.0` does not contain MSBuild.  otherwise we will need to wait for dev16.0 images to take this change.